controllers/mysql_conf.go

```diff
@@ -0,0 +1,178 @@
+package controllers
+
+import (
+	"fmt"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/cybozu-go/moco"
+)
+
+var (
+	// Default options of mysqld section
+	defaultMycnf = map[string]string{
+		"tmpdir":                              moco.TmpPath,
+		"innodb_tmpdir":                       moco.TmpPath,
+		"character_set_server":                "utf8mb4",
+		"collation_server":                    "utf8mb4_unicode_ci",
+		"skip_character_set_client_handshake": "ON",
+
+		"default_time_zone": "+0:00",
+
+		"back_log":            "900",
+		"max_connections":     "5000",
```

Comment:
I think a larger value, for instance, `100000` is safer/better because 5,000 has no good reasons.